scripts/test/system.sh

```diff
@@ -186,8 +187,50 @@ _subctl benchmark throughput "${KUBECONFIGS_DIR}"/kind-config-cluster1 "${KUBECO
 _subctl cloud prepare generic --context cluster1
 # Deprecated variant
 _subctl cloud prepare generic --kubecontext cluster1 && exit 1
+comment
+
+# Rename old broker-info.subm to be able to compare it with the recovered broker-info.sum
+echo "Renaming broker-info.subm to broker-info.subm.orig"
+mv "${DAPPER_SOURCE}"/output/broker-info.subm "${DAPPER_SOURCE}"/output/broker-info.subm.orig
+
+# All the tests assume that the current context is cluster2
+# Please refer the test matrix sheet for better understanding
+# https://docs.google.com/spreadsheets/d/1a4Ufw1EqBcms5FfDPx9zK_ep7zw77__OvCWyr_M6jJc/edit#gid=0
+
+# Test subctl recover-broker-info invocations which will always fail (no matter the broker and Submariner installations and the current
+# context set) with broker not found error # all blues
+_subctl recover-broker-info --namespace non-existent && exit 1
+_subctl recover-broker-info --brokercontext cluster2 && exit 1
+_subctl recover-broker-info --context cluster2 && exit 1
+
+# Test subctl recover-broker-info invocations with Submariner installed on both or just Broker cluster
+_subctl recover-broker-info --brokercontext cluster1 --context cluster1
+cmp "${DAPPER_SOURCE}"/output/broker-info.subm.orig broker-info.subm
+rm -f broker-info.subm
+
+_subctl recover-broker-info --brokercontext cluster1 --context cluster2
+cmp "${DAPPER_SOURCE}"/output/broker-info.subm.orig broker-info.subm
+rm -f broker-info.subm
```

Comment:
We can remove this case as the test args are redundant with L223 below but the behavior is actually the same as L207 above. Since Submariner is installed on the Broker, the `--context` are isn't used.